doc/man5/config.pod

```diff
@@ -486,6 +487,54 @@ built-in entropy sources will be used.
 
 =back
 
+=head2 ASN.1 String-Table Configuration
+
+The name B<stbl_section> in the initialization section names the section
+containing overrides for the built-in ASN.1 string-table size limits
+(defined in C<tbl_standard.h>). Within this section, each entry is a
+name/value pair where the B<name> is an ASN.1 object short or long name,
+and the B<value> is a comma-separated list of parameters in the form
+C<min:n>, C<max:n>, C<mask:mask>, or C<flags:flags>.
+
+=over 4
+
+=item B<min and max>
+
+Specify the minimum and maximum character counts, respectively.
+
+=item B<mask>
+
+Passed through L<ASN1_str2mask(3)>; defines which ASN.1 string encodings
+are permitted (for example: printableString, ia5String, utf8String,
+bmpString, numericString, teletexString, etc.).
+
+=item B<flags>
+
+Accepts exactly two values:
+
+=over 4
+
+=item B<nomask>
+
+disable all mask checking (STABLE_NO_MASK)  
```

Comment:
This is not the most appropriate description. What `nomask` does is to disable application of the "global mask" which restrictions which encodings are allowed. Without `nomask`, the specified encodings are further filtered by dropping any that are outside the global mask.